src/ghdcbot/adapters/storage/sqlite.py

```
@@ -950,6 +965,162 @@ def update_social_profile_display(
+        self,
+        repo: str,
+        pr_number: int,
+        channel_id: str,
+        message_id: str,
+        status: str = "open",
+        events: list[dict] | None = None,
+        pr_title: str = "",
+        author_github: str = "",
+    ) -> None:
+        """Save a new tracked PR channel message announcement."""
+        now = datetime.now(timezone.utc).isoformat()
```

Ruff (0.16.2)

[warning] Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)